src/cryptography/hazmat/primitives/serialization/base.py

```diff
@@ -21,10 +21,10 @@ def load_pem_private_key(
     *,
     unsafe_skip_rsa_key_validation: bool = False,
 ) -> PrivateKeyTypes:
```

Comment:
Is there any reason to keep the wrapper functions here rather than simply doing `load_pem_private_key = rust_openssl.keys.load_pem_private_key`?